test/test_meta.py

```diff
@@ -1327,6 +1330,53 @@ def test_embedding_bag_byte_unpack(self):
             self.assertEqual(res.dtype, torch.float32)
             self.assertEqual(res.untyped_storage().data_ptr(), 0)
 
+    def test_meta_function_with_out(self):
```

Comment:
What I meant is that you could test all the ops in PyTorch to make sure we're setting the `out=` name correctly in all of them. You have a standalone test similar to `run_meta_crossref`, or you can even test this in `run_meta_crossref`. You'll also need to put together a list of xfails (or fix) for those functions that do not have their names right.

Reply:
Ah I'm with you. Yesterday I found out that a colleague at Graphcore indepently found the same issue via decompositions rather than meta (shape inference): #107707 but has added tests which I think are what you suggest. I'll ask tests can be added here instead.

Reply:
Right. Do you know this person? If you do, could you coordinate with them to see how to land these PRs, as they are in conflict.

I prefer your solution, but it's true that they have those tests. If they're alright with it, you could carve out the tests form their PR and add him as a coauthor in this PR?

Reply:
Yes, different offices in UK but we are working on it now.